deployment/group_vars/dai.yml

```diff
@@ -39,3 +39,12 @@ UI_HOME_EXPLORER_ADDRESS_TEMPLATE: https://blockscout.com/poa/dai/address/%s
 UI_FOREIGN_EXPLORER_ADDRESS_TEMPLATE: https://blockscout.com/eth/mainnet/address/%s
 UI_HOME_GAS_PRICE_FALLBACK: 1000000000
 UI_FOREIGN_GAS_PRICE_FALLBACK: 10000000000
+
+## Monitor
+MONITOR_PORT: 3003
```

Comment:
Monitor port here is different from the rest of networks where it's 3001

Reply:
Good catch. Actually I wanted monitor ports for all bridges to be 3003, different than UI's 3001. Updated now.